start.sh

```diff
@@ -1,28 +1,16 @@
 #!/bin/bash
 
-if [[ -f /settings/uwsgi.ini ]]; then
-  rm /settings/uwsgi.ini
-fi;
-cat > /settings/uwsgi.ini <<EOF
-[uwsgi]
-chdir = /mapproxy
-pyargv = /mapproxy.yaml
-wsgi-file = app.py
-pidfile=/tmp/mapproxy.pid
-http = 0.0.0.0:8080
-processes = $PROCESSES
-cheaper = 2
-enable-threads = true
-threads = $THREADS
-master = true
-wsgi-disable-file-wrapper = true
-memory-report = true
-harakiri = 60
-chmod-socket = 664
-uid = 1000
-gid = 10001
-EOF
-
+# Check if uwsgi configuration exists
+if [[ ! -f /settings/uwsgi.ini ]]; then
+  echo "/settings/uwsgi.ini doesn't exists"
+  # If it doesn't exists, copy from /mapproxy directory if exists
+  if [[ -f /mapproxy/uwsgi.ini ]]; then
+    cp -f /mapproxy/uwsgi.ini /settings/uwsgi.ini
+  else
+    # default value
+    envsubst < /settings/uwsgi.default.ini > /settings/uwsgi.ini
+  fi
+fi
```

Comment:
Motivation for this is one:

We can let the user use their own `uwsgi.ini` config file. So, do not remove the file automatically.
The possible approach for the user to supply the config:
- docker-compose user in prod mode can supply the config from their volume in `/mapproxy/uwsgi.ini`
- docker-compose user in dev mode can supply the config by mounting the config in `/settings/uwsgi.ini`
- k8s user can use configmap and mount the config in `/settings/uwsgi.ini`
- k8s user with templating engine (like helm) can use configmap with template mounted in `/settings/uwsgi.default.ini` in cases where the parameter is supplied in the pod's environments/secrets.